TESTS/mbed_platform/FileHandle/main.cpp

```diff
@@ -420,10 +420,10 @@ void test_fseek_ftell()
     TestFile<FS>::resetFunctionCallHistory();
     fssek_ret = std::fseek(file, 0, SEEK_SET);
     TEST_ASSERT_EQUAL(0, fssek_ret);
+    TEST_ASSERT_TRUE(TestFile<FS>::functionCalled(TestFile<FS>::fnSeek));
```

Comment:
It seems like an optimization, not calling seek if you are already there....

Reply:
Ah, could be. Already pushed another attempt which should clear it, covering that possibility too. It'll have to call seek for the `SEEK_END`.